mmdet/core/bbox/assigners/hungarian_assigner.py

```diff
@@ -42,13 +42,16 @@ def __init__(self,
                  bbox_weight=1.,
                  iou_weight=1.,
                  iou_calculator=dict(type='BboxOverlaps2D'),
-                 iou_mode='giou'):
+                 iou_mode='giou',
+                 focal_loss=dict(alpha=0.25, gamma=2.0),
```

Comment:
comma is unnecessary.

Reply:
Haved updated, use `build_match_cost` to build all loss, please review